### DISCUSSION TITLE
Notarized version as up-to-date as the non-notarized?

### DISCUSSION THREAD

**Discussion by sbocconi** [chris1111/Wireless-USB-Big-Sur-Adapter#251] [Q&A] [answered]
Thank you very much for your work!!!

Is the notarized version as up-to-date as the non-notarized?

[Notarized Wireless USB Big Sur Adapter.zip](https://github.com/chris1111/Wireless-USB-Big-Sur-Adapter/files/7713890/Notarized.Wireless.USB.Big.Sur.Adapter.zip)

I wanted to avoid disabling SIP, I am on MacOS Monterey 12.6.1.

Thanks

**Comment by chris1111** [comment]
Hi @sbocconi  Your certainly missing some thing because the guide work, and the notarized kext version is the same as the Installer.
Anyway you have open dissussion but it look like an issue from your side
Look at my Today test at 5:21-h PM
![Screen Shot 2022-12-09 at 5 21 49 PM](https://user-images.githubusercontent.com/6248794/206805964-6473ab94-4031-4ca2-977e-e8fc4d4f9baa.png)


It work even in Ventura 13

![Screenshot 2022-12-09 at 5 53 38 PM](https://user-images.githubusercontent.com/6248794/206809938-af1cd6dd-a1d3-4d82-af4f-1a6f025cf159.png)

**Reply by sbocconi** [reply]
Thanks, I do not have any issues, I thought that the zip file was older than the latest changes you report on the web.

With the notarized version I guess I do not need to do:
`sudo spctl --master-disable`

but only

`/usr/sbin/spctl kext-consent add ZYM2ETK3E7`

Right?

**Comment by chris1111** [comment, marked as answer]
Yes or
`spctl kext-consent add ZYM2ETK3E7`
